jekyll.gemspec

```diff
@@ -38,6 +38,6 @@ Gem::Specification.new do |s|
   s.add_runtime_dependency("liquid",                "~> 4.0")
   s.add_runtime_dependency("mercenary",             "~> 0.3.3")
   s.add_runtime_dependency("pathutil",              "~> 0.9")
-  s.add_runtime_dependency("rouge",                 "~> #{ENV["ROUGE_VERSION"] || "1.7"}")
+  s.add_runtime_dependency("rouge",                 "~> #{ENV["ROUGE_VERSION"] || "2.1"}")
```

Comment:
This is a big wonky IMO.  You could probably do something like:

```ruby
s.add_runtime_dependency "rogue", ">= 1.7", "< 2.2"
```

And have a much easier time.